installer-tests/run_tests.py

```diff
@@ -0,0 +1,155 @@
+import glob
+import os
+import pexpect
+import subprocess
+import re
+
+def _expect(line, current_cmd, do_re_escape=True, do_detect_slow=True, strip_comments=True,
+            verbose=True):
+    timeout = 1
+    if do_detect_slow:
+        if line.startswith('$[slow]'):
+            print 'Slow line...'
+            timeout = 30
+            line = line.replace('$[slow]', '', 1)
+
+    if verbose:
+        print 'expecting', line
+
+    if do_re_escape:
+        line = re.escape(line)
+
+    current_cmd.expect(line, timeout=timeout)
+
+
+TEST_ROOT=os.path.dirname(os.path.abspath(__file__))
+SANDSTORM_SOURCE_TREE=os.path.abspath(os.path.join(os.path.abspath(__file__), '..', '..'))
+SANDSTORM_IN_HOME=os.path.join(os.environ['HOME'], 'sandstorm')
+
+def vagrant_destroy():
+    subprocess.check_output(['vagrant', 'destroy', '-f'], cwd=TEST_ROOT)
+
+def handle_test_script(lines):
```

Comment:
I like the idea of there being a `TestScript`. I will probably adjust that in the next revision.